src/exchange_solver.h

```diff
@@ -5,18 +5,33 @@
 
 namespace cyclus {
 
+class Context;
 class ExchangeGraph;
+class Arc;
 
 /// @class ExchangeSolver
 ///
 /// @brief a very simple interface for solving translated resource exchanges
 class ExchangeSolver {
  public:
-  explicit ExchangeSolver(bool exclusive_orders = false)
+  /// default value to allow exclusive orders or not
+  static const bool kDefaultExclusive = true;
+  
```

Comment:
In practice, this constructor is always called with an argument. This change matches the code default to the XML-input default.